joint_state_controller:
 type: joint_state_controller/JointStateController
 publish_rate: 50


x_position_controller:
 type: position_controllers/JointPositionController
 joint: move_x
 pid: {p: 100.0, i: 50.0, d: 10.0}
y_position_controller:
 type: position_controllers/JointPositionController
 joint: move_y
 pid: {p: 100.0, i: 50.0, d: 10.0}
z_position_controller:
 type: position_controllers/JointPositionController
 joint: move_z
 pid: {p: 100.0, i: 50.0, d: 10.0}

